guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,172 @@
+- **Facets** are a subset of filtering. Facets are document fields used as categories and thus provide grouping capabilities to search for specific fields rather than every field. They allow users to narrow down a set of documents by multiple dimensions at a time.
+
+Faceting and filtering aim at being complementary.
+
+## Setting Up Facets
+
+Document attributes to use as facets **must be declared at indexing time**.
+
+You can set up facets **through the API** via the [global settings route](/references/settings.md#update-settings).
+You need to add the desired attributes to the `attributesForFaceting` list. This attribute accepts an array of strings that specifies which attributes must be used as facets, and defaults to `null`.
+
+`attributesForFaceting=[<Attribute>, ...]`
+
+- `[<Attribute>, ...]` (Array of strings, defaults to `null`)
+
+  An array of strings that contains the attributes to use as facets.
+
+::: warning
+
+Only fields of data type **string** or **array of strings** can be set up as facets.
+
+:::
+
+Any POST request on the `settings` route with a value set to `attributesForFaceting` will overwrite the current faceted attributes. Then, passing an empty array will reset all defined faceted attributes.
+
+### Example
+
+Suppose that you have a collection of movies in the following JSON format:
+
+```json
+[
+  {
+      "id": "495925",
+      "title": "Doraemon the Movie:Nobita's Treasure Island",
+      "director": "Fujiko Fujio",
+      "genre": "Science fiction",
+      "poster": "https://image.tmdb.org/t/p/w1280/cmJ71gdZxCqkMUvGwWgSg3MK7pC.jpg",
+      "overview": "The story is based on Robert Louis Stevenson's Treasure Island novel.",
+      "release_date": 1520035200
+  },
+  {
+      "id": "329996",
+      "title": "Dumbo",
+      "director": "Tim Burton",
+      "genre": "Science fiction",
+      "poster": "https://image.tmdb.org/t/p/w1280/279PwJAcelI4VuBtdzrZASqDPQr.jpg",
+      "overview": "A young elephant, whose oversized ears enable him to fly, helps...",
+      "release_date": 1553644800
+  },
+  {
+      "id": "458723",
+      "title": "Us",
+      "director": "Jordan Peele",
+      "genre": "Horror",
+      "poster": "https://image.tmdb.org/t/p/w1280/ux2dU1jQ2ACIMShzB3yP93Udpzc.jpg",
+      "overview": "Husband and wife Gabe and Adelaide Wilson take their...",
+      "release_date": 1552521600
+  },
+  ...
+]
+```
+
+You would declare faceted attributes as follows:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "attributesForFaceting": [
+          "director",
+          "genre"
+      ]
+  }'
+```
+
+## Querying On Faceted Attributes
+
+When performing a search, you can specify:
+
+### 1. The facets for which to retrieve the matching count
+
+`facets=[<facetName>, <facetName>, ...]`
+
+This attribute can take two values:
+
+- `[<facetName>, <facetName>, ...]` (Optional, array of strings, defaults to `null`)
```

Comment:
What does this mean? I don't understand this sentence.

Reply:
Yes, you're right. I made some changes to make it more clear!